package.json

```
@@ -8,14 +8,15 @@
     "renovate-config-validator": "dist/config-validator.js"
   },
   "scripts": {
-    "build": "run-s clean 'generate:*' 'compile:*' create-json-schema",
+    "build": "run-s clean 'generate:*' create-protos 'compile:*' create-json-schema",
     "build:docker": "ts-node tools/docker.ts",
     "build:docs": "ts-node tools/generate-docs.ts",
     "clean": "rimraf dist tmp",
     "clean-cache": "node tools/clean-cache.mjs",
     "compile:ts": "tsc -p tsconfig.app.json",
     "config-validator": "ts-node lib/config-validator.ts",
     "create-json-schema": "ts-node tools/generate-schema.ts && prettier --write --cache 'renovate-schema.json'",
+    "create-protos": "ts-node tools/generate-protos.ts",
```

```suggestion
    "generate:create-protos": "ts-node tools/generate-protos.ts",
```
then all other changes can be reverted

---

See note in the other comment about generate not running in the lint pipeline.